lib/Statistics/Track/WooCommerceOrders.php

```diff
@@ -0,0 +1,43 @@
+<?php
+namespace MailPoet\Statistics\Track;
+
+use MailPoet\Models\StatisticsClicks;
+use MailPoet\Models\StatisticsWooCommerceOrders;
+use MailPoet\Models\Subscriber;
+use MailPoet\WooCommerce\Helper;
+use WC_Order;
+
+if (!defined('ABSPATH')) exit;
+
+class WooCommerceOrders {
+  const USE_CLICKS_SINCE_DAYS_AGO = 14;
+
+  /** @var Helper */
+  private $woocommerce_helper;
+
+  function __construct(Helper $woocommerce_helper) {
+    $this->woocommerce_helper = $woocommerce_helper;
+  }
+
+  function trackPaidOrder($id) {
+    $order = $this->woocommerce_helper->wcGetOrder($id);
+    if (!$order instanceof WC_Order || floatval($order->get_total()) <= 0) {
```

Comment:
Why do you think we shouldn't track orders with 0 total? I don't see it in the specification and I am not even sure what the use case is so I am asking, just to be sure.

Reply:
Hmm, good point. I though "free items don't participate on revenues" but maybe I should ask.

Reply:
Turns out we want to track 0 purchases as well.